frontend/gpt4all.io/src/components/models.jsx

```diff
@@ -1,88 +1,72 @@
 import React, { useState, useEffect } from "react";
 import * as DOMPurify from "dompurify";
 import { ArrowBigDownDash } from "lucide-react";
-import {
-    Select,
-    SelectContent,
-    SelectGroup,
-    SelectItem,
-    SelectLabel,
-    SelectTrigger,
-    SelectValue,
-} from "./ui/select";
 
 const getModels = async () => {
-    const response = await fetch("https://gpt4all.io/models/models.json");
-    const jsonData = await response.json();
-    return jsonData;
-}
+  //const response = await fetch("https://gpt4all.io/models/models.json");
+  const response = await fetch(
+    "https://raw.githubusercontent.com/nomic-ai/gpt4all/main/gpt4all-chat/metadata/models.json"
```

Comment:
I want to retain this url: https://gpt4all.io/models/models.json

It redirects to the one you posted.